cmd/collector/app/span_processor.go

```diff
@@ -126,10 +128,18 @@ func newSpanProcessor(spanWriter spanstore.Writer, opts ...Option) *spanProcesso
 	return &sp
 }
 
-// Stop halts the span processor and all its go-routines.
+// Stop is an alias to #Close(). Deprecated.
 func (sp *spanProcessor) Stop() {
```

Comment:
let's remove it. It's a method on a private receiver anyway, not part of a public API.